internal/engine/wazevo/ssa/instructions.go

```diff
@@ -913,8 +913,8 @@ var instructionReturnTypes = [opcodeEnd]returnTypesFn{
 	OpcodeVbnot:      returnTypesFnV128,
 	OpcodeVbandnot:   returnTypesFnV128,
 	OpcodeVbitselect: returnTypesFnV128,
-	OpcodeVanyTrue:   returnTypesFnV128,
-	OpcodeVallTrue:   returnTypesFnV128,
+	OpcodeVanyTrue:   returnTypesFnSingle,
+	OpcodeVallTrue:   returnTypesFnSingle,
```

Comment:
what's the rationale for these changes

Reply:
why can't you use `returnTypesFnI32`

Reply:
I did not notice there was a proper `returnTypesFnI32`